### DISCUSSION TITLE
Compilation for CEF

### DISCUSSION THREAD

**Discussion by Eriter555** [Eloston/ungoogled-chromium#3058] [Ideas]
Please compile the binaries for cef if it's not too much trouble. Specifically for Windows 64. Thanks.

**Comment by networkException** [comment]
I'm not familiar with CEF but I assume it requires pulling in sources for a wrapper in addition to chromium + ungoogled patches.

Generally, binaries are provided as a sort of side-product of the project (maintained by individuals, probably only with the motivation to be able to use a binary themselves). Your best chance of success would be to add the necessary steps to our [Windows CI](https://github.com/ungoogled-software/ungoogled-chromium-windows/tree/master/.github/workflows) yourself
